libclingo/clingo.h

```diff
@@ -3100,6 +3129,17 @@ CLINGO_VISIBILITY_DEFAULT bool clingo_control_theory_atoms(clingo_control_t cons
 //! @param[in] data user data passed to the observer functions
 //! @return whether the call was successful
 CLINGO_VISIBILITY_DEFAULT bool clingo_control_register_observer(clingo_control_t *control, clingo_ground_program_observer_t const *observer, bool replace, void *data);
+//! Register a backend with the control object.
+//!
+//! This function is similar to clingo_control_register_observer except that
+//! clingo's internal backends can be selected.
+//!
+//! @param[in] control the target
+//! @param[in] type the kind of backend to register
+//! @param[in] file the file to write the result to
+//! @param[in] replace just pass the grounding to the backend but not the solver
+//! @return whether the call was successful
```

Comment:
Shouldn't this mention similar error codes as `clingo_control_load_aspif` (or at least clingo_error_bad_alloc)?